openslides_backend/permissions/permission_helper.py

```diff
@@ -119,6 +119,32 @@ def has_committee_management_level(
     return False
 
 
+def get_shared_committee_management_levels(
+    datastore: DatastoreService,
+    user_id: int,
+    expected_level: CommitteeManagementLevel,
+    committee_ids: list[int],
+) -> list[int]:
+    """Checks wether a user has the minimum necessary CommitteeManagementLevel"""
+    if user_id > 0:
+        cml_fields = ["committee_management_ids"]
+        user = datastore.get(
+            fqid_from_collection_and_id("user", user_id),
+            [*cml_fields],
+            lock_result=False,
+            use_changed_models=False,
+        )
+        if user.get("organization_management_level") in (
```

Comment:
you forget to read the oml and additionally you should have a test to detect this

Reply:
1. The oml is read right in this line at `user.get("organization_management_level")
2. This code is in case the user is a super- or orga-admin because in this case he is implicitly also allowed to above any committee_admin for the sake of the user_scope